.bazelrc

```diff
@@ -1 +1,7 @@
 build --host_platform=@io_tweag_rules_nixpkgs//nixpkgs/platforms:host
+build --crosstool_top=@nixpkgs_config_cc//:toolchain
+# Using toolchain resolution can lead to spurious dependencies on
+# `@local_config_cc//:builtin_include_directory_paths`. This needs to be
```

Comment:
@aherrmann is there a ticket tracking this? Once this flag is enabled, do I understand right that `--crosstool_top` will no longer be necessary?

Reply:
> @aherrmann is there a ticket tracking this?

I've looked into the issue in the daml repo once more. It looks like this is a rules_haskell issue: https://github.com/tweag/rules_haskell/issues/1467

However, in general https://github.com/bazelbuild/bazel/issues/1258 suggests that there is still ongoing work in Bazel in this area. And https://github.com/bazelbuild/bazel/issues/12712 also seems related.

> Once this flag is enabled, do I understand right that --crosstool_top will no longer be necessary?

Correct, assuming that all involved Starlark code is compatible with cc toolchain resolution. E.g. any direct reference to `ctx.attr._cc_toolchain` breaks this and causes a dependency on `--crosstool_top`.